pkg/testpackage/testpackage.go

```diff
@@ -3,7 +3,7 @@ package testpackage
 import (
 	"flag"
 	"regexp"
-	"strings"
+  "strings"
```

Comment:
Please use tab, not spaces.
```suggestion
	"strings"
```